apps/emqx_connector/src/emqx_connector_jwt.erl

```diff
@@ -0,0 +1,46 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2022 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+
+-module(emqx_connector_jwt).
```

Comment:
NIT: from the module name couldn't it be assumed that it is some kind of `emqx_resourse`behaviour implementation?

Reply:
This module is solely for encapsulating the API for reading the JWTs (from this [feedback](https://github.com/emqx/emqx/pull/9241#discussion_r1009394481)).